pkg/pool-stable-phantom/contracts/StablePhantomPool.sol

```diff
@@ -29,7 +29,7 @@ import "@balancer-labs/v2-pool-utils/contracts/BaseGeneralPool.sol";
 import "@balancer-labs/v2-pool-utils/contracts/rates/PriceRateCache.sol";
 import "@balancer-labs/v2-pool-utils/contracts/ProtocolFeeCache.sol";
 
-import "./StableMath.sol";
```

Comment:
Seems like we accidentally removed this

Reply:
It's included by StablePoolStorage

Reply:
Eh I'd still have it here, since we rely on it heavily. There's no harm in doing a double include.